### PR TITLE
Release v0.18.0 QA

### PR DESCRIPTION
**Pull request by Ajarmar**
**What this PR does / why we need it**: Fixes from the QA process for release v0.18.0 of apps.

**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes #

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [X] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [ ] I upgraded no Chart.
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

**Review by crssnd — APPROVED** [review]

File: config/config/sc-config.yaml [hunk 1]
```diff
@@ -166,24 +166,24 @@ harbor:
...
   notarySigner:
     resources:
       requests:
         cpu: 2m
         memory: 10Mi
       limits:
-        cpu: 5m
-        memory: 20Mi
+        cpu: 100m
+        memory: 250Mi
```
Comment (viktor-f) [inline config/config/sc-config.yaml]: Some of these are quite big changes in limits (this is just one of them). Should we also increase the requests a bit?

Comment (robinAwallace) [reply config/config/sc-config.yaml]: It is probably a good idea to increase the requests as well. To ensure that the pods can run when some load are put on them. Unsure what the best practices here are, should requests and limits always follow each other?

Comment (viktor-f) [reply config/config/sc-config.yaml]: I'd say that we don't have any strict requirements here. We don't always set request=limit, so you don't have to either.
You can use your judgement to set them to something sensible, if you have any data from when you wanted to increase the limits, then maybe base it on that.

Comment (crssnd) [reply config/config/sc-config.yaml]: personally I set the limit = 2 x request

**Pull request merged**